### PR TITLE
96boards-tools: Set ConditionFirstBoot=yes

### PR DESCRIPTION
**Pull request by kraj**
So it runs only on first boot

Change-Id: If03d157f39885bdb957a73b7d4da5cefca24fc32
Signed-off-by: Khem Raj <raj.khem@gmail.com>

### CODE REVIEW

File: recipes-bsp/96boards-tools/files/0001-resize-helper.service-Use-ConditionFirstBoot-directi.patch [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ [Unit]
+ Description=Resize root filesystem to fit available disk space
+ After=systemd-remount-fs.service
++ConditionFirstBoot=yes
+ 
+ [Service]
+ Type=oneshot
+ ExecStart=-/usr/sbin/resize-helper
+-ExecStartPost=/bin/systemctl disable resize-helper.service
++ExecStartPost=/bin/systemctl reboot
```
Comment (ndechesne) [inline recipes-bsp/96boards-tools/files/0001-resize-helper.service-Use-ConditionFirstBoot-directi.patch]: why is that needed? this is quite a change compared to the current behavior.. for most of the users a reboot will put the board back into fastboot, so it's not convenient..

Comment (kraj) [reply recipes-bsp/96boards-tools/files/0001-resize-helper.service-Use-ConditionFirstBoot-directi.patch]: for filesystem consistency reboot is needed. I have seen failures in webkit when contrinuing to use the system. Then I checked debian on rpi and they do reboot automatically after resize as well.

Comment (ndechesne) [reply recipes-bsp/96boards-tools/files/0001-resize-helper.service-Use-ConditionFirstBoot-directi.patch]: well, the fact that they do it for rpi doesn't make a good argument to me.. is there a bug report or anything that we can rely on? as far as i know, ext4 can be resized while mounted.. forcing a reboot at this point doesn't seem right to me.. 

[and yes, I would prefer to have an initrd and do it from there, before mounting rootfs..]

Comment (kraj) [reply recipes-bsp/96boards-tools/files/0001-resize-helper.service-Use-ConditionFirstBoot-directi.patch]: I dont have reference to bugs to cite here but I think current approach is equally problematic ( i.e. live resize ) that reboot or not would not matter as much we need to reload the partition table after fdisking and I see that we are calling partprobe for this and then resize2fs is called so apps get the revised size when probed.

I will send a v2 without causing a reboot.

**Comment by kraj** [comment]
proposed with another request
